### ISSUE TITLE
[Bug]: the vm_ids  parameter of `cloudavenue_vm_affinity_rule` is limited at 2 values only

### ISSUE THREAD

**Issue by smolinari-obs** [orange-cloudavenue/terraform-provider-cloudavenue#380]
### Terraform Core Version

1.5.1

### Cloud Avenue Provider Version

0.5.2

### Affected Resource(s)

* cloudavenue_vm_affinity_rule

### Expected Behavior

Add a list of VM ids with more than 2 entries.

### Actual Behavior

If we set more than two vm_ids we have an error message.

### Relevant Error/Panic Output Snippet

```shell
terraform plan
╷
│ Error: Invalid Attribute Value
│ 
│   with cloudavenue_vm_affinity_rule.affinity-rule,
│   on affinity-rule.tf line 7, in resource "cloudavenue_vm_affinity_rule" "affinity-rule":
│    7:   vm_ids = [
│    8:     "urn:vcloud:vm:601b7f4b-a9a8-409e-bbd7-0304259e1a1a",
│    9:     "urn:vcloud:vm:cb539230-903e-41fc-a257-ecc8911aabee",
│   10:     "urn:vcloud:vm:bcad1915-5be6-45b1-9936-77833df6524d"
│   11:   ]
│ 
│ Attribute vm_ids list must contain at most 2 elements, got: 3
```

### Terraform Configuration Files
```hcl
resource "cloudavenue_vm_affinity_rule" "affinity-rule" {
  name     = "TestAffinityRule"
  polarity = "Anti-Affinity"
  enabled = true
  required = false
  vdc = local.cav_vdc
  vm_ids = [
    "urn:vcloud:vm:601b7f4b-a9a8-409e-bbd7-0304259e1a1a",
    "urn:vcloud:vm:cb539230-903e-41fc-a257-ecc8911aabee",
    "urn:vcloud:vm:bcad1915-5be6-45b1-9936-77833df6524d"
  ]
}
```

### Steps to Reproduce

Create an cloudavenue_vm_affinity_rule ressource with more than 2 vm_ids.

### Would you like to implement a fix?

None

**Comment by azrod** [comment]
Hi @smolinari-obs 👋 Thank you for reporting these issues and sorry you are running into this unexpected behavior.
We support this issue and will let you know when the fix is ​​released.

**Comment by azrod** [comment]
### 🚀 🎉 Release V0.6.0
Your bug report has been fixed in release [v0.6.0](https://github.com/orange-cloudavenue/terraform-provider-cloudavenue/releases/tag/v0.6.0)
